.travis.yml

```diff
@@ -92,12 +99,6 @@ matrix:
                CONDA_CHANNELS='astropy-ci-extras astrofrog'
                CONDA_DEPENDENCIES='PyQt5'
```

Comment:
@bsipocz - the issue is that this should actually be ``pyqt5`` (all lowercase). Not sure how it ended up uppercase and how it worked before.